_data/CRefice/CRefice.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Carlo Refice
+institution: University of Groningen
+profile_pic: CRefice.jpg
+quote: "Programming is like a cup of morning coffee: it's comforting, stimulating, and it causes addiction."
```

Comment:
Please do not include quotes(")

Reply:
Done, though the example MonaTheOctocat .md file does use them.